sys/net/gnrc/network_layer/sixlowpan/frag/gnrc_sixlowpan_frag.c

```diff
@@ -55,7 +55,7 @@ static inline uint8_t _max_frag_size(gnrc_netif_t *iface,
     }
 #endif /* MODULE_GNRC_SIXLOWPAN_FRAG_HINT */
     (void)fbuf;
-    return iface->sixlo.max_frag_size;
+    return fbuf->best_frag_size ? fbuf->best_frag_size : iface->sixlo.max_frag_size;
```

Comment:
Any reason not to always use `best_frag_size` here?
It should be set to `max_frag_size` as a fallback anyway.